### PR TITLE
Configuring Azure Key Vault via Azure Web App

### PR DESCRIPTION
**Pull request by mattou07**
Another method to incorporate Key Vault into an Umbraco application running on an Azure Web App

### CODE REVIEW

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:9]:
```suggestion
From a security perspective, storing your application secrets in Azure Key Vault is always a good solution. This could be a connection string or other keys.

This article tells you how to configure your application so it is ready to use a Key Vault.
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:11]:
```suggestion
Depending on your hosting situation there are a few approaches to incorporating Azure Key Vault into your application.
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:14]:
```suggestion
1. For most Hosting scenarios: Install Key Vault via Nuget and retrieve your secrets directly in your Program.cs.
2. For Azure Web App Hosting: Directly reference a secret in your Azure Web App Configuration Application Settings.
```

Comment (sofietoft) [inline Extending/Key-Vault/index.md:14]: How about using this list as a table of content and link down to the related sections in the article?

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:80]:
```suggestion
Under **System assigned** change the Status from Off to **On**.
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:96]:
```suggestion
You will now be presented with different permissions to set for your Web App. You only need **Get** and **List** for **Secret Permissions** only. Click **Next** to continue:
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:139]:
```suggestion
Wait a moment and refresh the screen. You should see a Green tick. If you do not have a Green tick you need to review your Access Policies in the previous step.
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:160]:
```suggestion
On the Web App select **Advanced Edit** for Application Settings:
```

File: Extending/Key-Vault/index.md
Comment (sofietoft) [inline Extending/Key-Vault/index.md:163]:
```suggestion
Paste in the following JSON Objects inside the square brackets. Ensure you update `{keyvault-name}`, `{secret-name}` and `{version-id}`.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for the PR @mattou07 ! 💪 Really appreciate it!

I've added some comments and suggestions - mainly to the grammar and formatting of the article.
I'll send the article by our developers as well, so we can get all the technical stuff verified 💪

File: Extending/Key-Vault/index.md
Comment (mattou07) [reply Extending/Key-Vault/index.md:14]: Could you give me another page on Our that does this? Not sure what the syntax would look like in Markdown.

File: Extending/Key-Vault/index.md
Comment (mattou07) [reply Extending/Key-Vault/index.md:160]: I think its important to highlight that names need double underscore in its name to reference nested JSON properties in Azure. Unless you think the example is clear enough. I like to reiterate 😄

Comment (sofietoft) [reply Extending/Key-Vault/index.md:160]: Oh! I'm not suggestion to remove anything, simply changing "click" to "select".
Not sure why it's highlighting some of the text in read here 🤔

File: Extending/Key-Vault/index.md
Comment (mattou07) [reply Extending/Key-Vault/index.md:73]: Done

File: Extending/Key-Vault/index.md
Comment (mattou07) [reply Extending/Key-Vault/index.md]: Done

File: Extending/Key-Vault/index.md
Comment (Jeavon) [inline Extending/Key-Vault/index.md:72]: As this feature is called "Key Vault references" I would recommend using that in the title, perhaps "Use Key Vault references for Azure App Services"

Comment (Jeavon) [reply Extending/Key-Vault/index.md:72]: Also perhaps add a link to https://learn.microsoft.com/en-us/azure/app-service/app-service-key-vault-references

Comment (mattou07) [reply Extending/Key-Vault/index.md]: Thanks Jeavon, this is now updated.

**Comment by mattou07** [comment]
@sofietoft Just double checking, is there any thing left I need to do or is it queued to be reviewed by the dteam?

**Review by Zeegaan — APPROVED** [review]
LGTM 👍

**Comment by sofietoft** [comment]
Hi @mattou07 !
We **were** waiting for a review from the d-team, but it looks like we've gotten that now 👏 
Thanks @Zeegaan 💪 Much appreciated! 

I'll see to it, that this is merged in 😄

**Review by sofietoft — APPROVED** [review]

**Pull request merged**
